packages/@ourworldindata/components/src/TextWrap/TextWrap.tsx

```diff
@@ -340,4 +319,12 @@ export class TextWrap {
             </text>
         )
     }
+
+    render(
+        x: number,
+        y: number,
+        options: SVGRenderProps = {}
+    ): React.ReactElement {
+        return this.renderSVG(x, y, options)
+    }
```

Comment:
I'd rather keep it at `renderSVG`, `renderHTML` for these, just to make it explicit at the call site what the expected output is.

Reply:
Not sure why I added this..

Reply:
Ah, now I realise it always had a `render` method – I just extracted it into `renderSVG`. I quickly scanned the codebase and we don't have any `<TextWrap />` calls, so I'll go ahead and remove it

Reply:
Yeah, it never was a proper React `render` method either way, as it _always_ needs its args passed :)